docs/v4/start/upgrade.md

```diff
@@ -39,6 +39,35 @@ $app = new App(...);
 
 ## Changes to Container
 Slim no longer has a Container so you need to supply your own. If you were relying on request or response being in the container, then you need to either set them to a container yourself, or refactor. Also, `App::__call()` method has been removed, so accessing a container property via `$app->key_name()` no longer works.
+```php
+/**
+ * Slim 3.x shipped with the Pimple container implementation and enabled the following synthax
+ */
+$container = $app->getContainer();
+
+//Assign dependencies as array
+$container['view'] = function (\Psr\Container\ContainerInterface $container){
+    return new \Slim\Views\Twig('');
+};
+
+
+/**
+ * Slim 4.x doesn’t come with a container. Slim supports containers that implement PSR-11 like PHP-DI.
```

Comment:
~~Slim 4.x doesn’t come with a container. Slim supports containers that implement PSR-11 like PHP-DI.~~

Slim 4.x does not ship with a container library. It supports all PSR-11 implementations such as PHP-DI

Reply:
Changes done!